src/controllers/TalksController.php

```diff
@@ -215,16 +211,11 @@ public function deleteAction($request, $db)
      */
     public function addTrackToTalk(Request $request, PDO $db)
     {
-        if (!isset($request->user_id)) {
-            throw new Exception("You must be logged in to create data", 400);
```

Comment:
We had a discussion about that a while back and decided to leave them at the moment as they are. If changed that should also be a separate PR that then changes *all* of those occurences throughout the API